src/querystringsafe_base64/__init__.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
```

Comment:
I don't like actual code (functions and classes) in `__init__.py`. To the point when I'm planning on writing a custom plugin for pylint for that.

If it fits in a single file, putting the code into 'src/querystringsafe_base64.py' would be better (because less nested).